package.json

```diff
@@ -5,6 +5,7 @@
     "@babel/preset-react": "^7.14.5",
     "@emotion/react": "^11.4.1",
     "@emotion/styled": "^11.3.0",
+    "@material-ui/core": "^4.12.3",
```

Comment:
```suggestion
```
I think the having both versions of material ui is what broke the header, remove this.